webclient/arch_z/templates/arch_z/dj/partials/komponenta_create.html

```diff
@@ -22,12 +22,5 @@
   </div>
 </div>
 <script src="{% static 'js/ajax_functions.js' %}"></script>
-<script>
-    document.addEventListener('DOMContentLoaded', () => {
-        show_form(`create_komponenta_form`);
-        url = '{% url "heslar:get-initial-value" %}' + "?nadrazene="
-        get_vychozi_hodnota_podrazeneho("#id_areal","#id_aktivity",url)
```

Comment:
Nepřestane tady fungovat to nastavování výchozí hodnoty? Nebo se to pak řeší jinde?

Reply:
Je potřeba to zachovat, udělal jsem drobný refactor.

Reply:
Ok, pokud úprava to fungování ostatních míst neovlivní, tak zvaž jestli je to třeba upravit, nebo ne. Pokud myslíš, že ano, udělej na to prosím zvlášť issue.